src/Aspire.Hosting.Azure/Provisioning/Provisioners/AzureProvisioner.cs

```diff
@@ -184,9 +184,16 @@ await UpdateStateAsync(r, s => s with
 
     private static async Task<JsonObject> GetUserSecretsAsync(string? userSecretsPath, CancellationToken cancellationToken)
     {
+        var jsonDocumentOptions = new JsonDocumentOptions
```

Comment:
nit: I know that `JsonSerializerOptions` should be static, maybe this one should be too. At least if it can it wouldn't hurt.

Reply:
Agree